assets/src/edit-story/components/canvas/multiSelectionMovable.js

```diff
@@ -149,11 +155,17 @@ function MultiSelectionMovable({ selectedElements }) {
   // Update elements once the event has ended.
   const onGroupEventEnd = ({ targets, isRotate, isResize }) => {
     const updates = {};
+    const toRemove = [];
     targets.forEach((target, i) => {
+      const { element, updateForResizeEvent } = targetList[i];
+      if (isTargetOutOfContainer(target, pageContainer.parentNode)) {
```

Comment:
The `.parentNode` reference here (and in `singleSelectionMovable`) is very fragile to changes in DOM structure. Could we place the reference on the right element somehow?

Reply:
I believe we have `fullbleedContainer` as well?